Assets/MixedRealityToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -21,7 +23,7 @@ namespace MixedRealityToolkit.SpatialMapping
     /// </summary>
     [RequireComponent(typeof(Collider))]
     [RequireComponent(typeof(Interpolator))]
-    public class TapToPlace : MonoBehaviour, IInputClickHandler
+    public class TapToPlace : MonoBehaviour, IInputClickHandler, IInputSource
```

Comment:
We probably don't need to make this an input source, but just pass the click input source when we raise the event.